test/apps/collision/app.jsx

```diff
@@ -8,7 +8,8 @@ import {GeoJsonLayer, SolidPolygonLayer, TextLayer} from '@deck.gl/layers';
 import {CollisionFilterExtension, MaskExtension} from '@deck.gl/extensions';
 import {VectorTileLayer, vectorTableSource} from '@deck.gl/carto';
 
-const accessToken = 'XXX';
+const accessToken =
```

Comment:
Do we really want to store access tokens in source?

Reply:
It is scoped to a single public dataset only